tools/dockerfile/test/python_stretch_3.8_x64/get_cpython.sh

```diff
@@ -1,29 +0,0 @@
-#!/bin/bash
```

Comment:
I got the intent, but I think it was actually misleading. The dockerfile looks like it's always going to test with the latest version of python 3.8 but it's not the actually the case in practice due to the way we build and refresh testing docker images. Both in kokoro and locally, the dockerfile will be stuck with some version of python 3.8 and won't get rebuilt until the dockerfile is manually modified anyway (to break docker cache locally or to allow re-pushing the image for kokoro).
A few additional reasons:
- i think generally, it's better to use fixed versions in tests because it makes things more predictable and means fewer unexpected breakages (which will usually be encountered first by someone who doesn't have enough context to solve them efficiently)
- the new version is more readable IMHO